### PR TITLE
Add FTP File Transfer Instructions to Wiki (Get and Put options)

### PR DESCRIPTION
**Pull request by leonteale**
This pull request adds a new section to the Wiki documentation, providing detailed instructions on using FTP commands for file access and transfer. The addition follows the existing documentation style and includes:

- Instructions for listing files in a directory, downloading, and uploading files using FTP.
- Command examples for each operation.
- A brief description at the beginning of the document for context.

The intent is to enhance the existing Wiki with FTP-related content, making it a more comprehensive resource for users needing guidance on various file transfer methods.

Please let me know if there are any changes or improvements you'd like me to make. I'm open to feedback and happy to collaborate to ensure the documentation is as helpful as possible for the community.

Best regards,
Leon Teale

### CODE REVIEW

File: ftp-protocol/get-and-put-files.md [hunk 1]
```diff
@@ -0,0 +1,55 @@
+---
+description: Instructions for using FTP commands to access and transfer files using NetExec.
+
```
Comment (NeffIsBack) [inline ftp-protocol/get-and-put-files.md:3]: I think here is `---` missing. Atleast the page doesnt get added to the gitbook when i try to preview the change

**Comment by NeffIsBack** [comment]
Thanks for the PR and the help on the wiki!

**Comment by NeffIsBack** [comment]
FYI here you can preview your changes:
image

**Review by leonteale — COMMENTED** [review]
updated

**Review by NeffIsBack — APPROVED** [review]
LGTM 👍

**Pull request merged**